src/databricks/labs/pytester/fixtures/unwrap.py

```diff
@@ -65,7 +65,7 @@ def __str__(self):
 _GENERATORS = set[str]()
 
 
-def call_stateful(some: Callable[..., T], **kwargs) -> tuple[CallContext, T]:
+def call_stateful(some: Callable[..., Generator[Callable[..., T]]], **kwargs) -> tuple[CallContext, T]:
```

Comment:
That's an eye-catching change. I wonder why the old value was accepted by the linting? 🤔

Reply:
I don't know either, maybe because the tests were not linted